### DISCUSSION TITLE
Realtime Subscriptions with Python

### DISCUSSION THREAD

**Discussion by arnausaumell** [supabase/supabase#25990] [Feature Requests]
Does Supabase python sdk have realtime table subscriptions enabled?

**Comment by obragaa** [comment]
Hii arnausaumell,

As of now, the Supabase Python SDK does not natively support realtime table subscriptions in the same way the JavaScript SDK does. However, you can work around this limitation by using the Supabase Realtime server directly through websockets. This allows you to listen to changes on your tables.

If you need this functionality, you might want to look into using the `supabase-py` library to set up a connection and then manage websocket connections to handle real-time updates. You can find examples and more detailed guidance on this in the Supabase documentation or community forums.

Hope this helps!
